driver-mct/shr/seq_infodata_mod.F90

```diff
@@ -782,6 +782,7 @@ SUBROUTINE seq_infodata_Init( infodata, nmlfile, ID, pioid, cpl_tag)
        infodata%iac_nx = 0
        infodata%iac_ny = 0
 
+
        infodata%nextsw_cday   = -1.0_SHR_KIND_R8
```

Comment:
is there a reason for all the extra spaces in this file? I didn't see any real modified code

Reply:
no- this is a mistake left over from some old modifications that I have since removed.